### PR TITLE
[bugfix] Added capability to read tn_netflow_monitor_pol_dn

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

File: client/l3extLIfP_service.go [hunk 1]
```diff
@@ -44,9 +44,7 @@ func (sm *ServiceManager) UpdateLogicalInterfaceProfile(name string, logical_nod
 }
 
 func (sm *ServiceManager) ListLogicalInterfaceProfile(logical_node_profile string, l3_outside string, tenant string) ([]*models.LogicalInterfaceProfile, error) {
-
-	baseurlStr := "/api/node/class"
-	dnUrl := fmt.Sprintf("%s/uni/tn-%s/out-%s/lnodep-%s/l3extLIfP.json", baseurlStr, tenant, l3_outside, logical_node_profile)
+	dnUrl := fmt.Sprintf("%s/%s/l3extLIfP.json", models.BaseurlStr, fmt.Sprintf(models.ParentDnl3extlifp, tenant, l3_outside, logical_node_profile))
```
Comment (akinross) [inline client/l3extLIfP_service.go:47]: "%s/%s/l3extLIfP.json" could point to model.L3extlifpClassName?

File: client/l3extLIfP_service.go [hunk 2]
```diff
@@ -142,8 +139,7 @@ func (sm *ServiceManager) DeleteRelationl3extRsPathL3OutAttFromLogicalInterfaceP
 }
 
 func (sm *ServiceManager) ReadRelationl3extRsPathL3OutAttFromLogicalInterfaceProfile(parentDn string) (interface{}, error) {
-	baseurlStr := "/api/node/class"
-	dnUrl := fmt.Sprintf("%s/%s/%s.json", baseurlStr, parentDn, "l3extRsPathL3OutAtt")
+	dnUrl := fmt.Sprintf("%s/%s/%s.json", models.BaseurlStr, parentDn, "l3extRsPathL3OutAtt")
```
Comment (akinross) [inline client/l3extLIfP_service.go:142]: this also has a model reference, models.L3extrspathl3outattClassName

**Review by akinross — APPROVED** [review]
LGTM

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**